### ISSUE TITLE
No action found since latest update 0.8.x

### ISSUE THREAD

**Issue by anthonymonori** [HazAT/fastlane-plugin-badge#1]
All my builds started suddenly failing since last night:

```
$ fastlane test --verbose
DEBUG [2017-03-14 10:51:40.85]: Checking if there are any plugins that should be loaded...
DEBUG [2017-03-14 10:51:40.91]: Loading 'fastlane-plugin-tpa' plugin
WARN [2017-03-14 10:51:41.13]: Installing Ruby gem 'fastlane-plugin-tpa'...
INFO [2017-03-14 10:51:48.68]: Successfully installed 'fastlane-plugin-tpa'
DEBUG [2017-03-14 10:51:48.71]: Loading 'fastlane-plugin-badge' plugin
WARN [2017-03-14 10:51:48.86]: Installing Ruby gem 'fastlane-plugin-badge'...
Building native extensions.  This could take a while...
ERROR [2017-03-14 10:51:57.97]: Error loading plugin 'fastlane-plugin-badge': ERROR: Failed to build gem native extension.

    current directory: /usr/local/bundle/gems/curb-0.9.3/ext
/usr/local/bin/ruby -r ./siteconf20170314-7-15s45ub.rb extconf.rb
checking for curl-config... no
checking for main() in -lcurl... no
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
	--with-opt-dir
	--without-opt-dir
	--with-opt-include
	--without-opt-include=${opt-dir}/include
	--with-opt-lib
	--without-opt-lib=${opt-dir}/lib
	--with-make-prog
	--without-make-prog
	--srcdir=.
	--curdir
	--ruby=/usr/local/bin/ruby
	--with-curl-dir
	--without-curl-dir
	--with-curl-include
	--without-curl-include=${curl-dir}/include
	--with-curl-lib
	--without-curl-lib=${curl-dir}/lib
	--with-curllib
	--without-curllib
extconf.rb:18:in `<main>':   Can't find libcurl or curl/curl.h (RuntimeError)

  Try passing --with-curl-dir or --with-curl-lib and --with-curl-include
  options to extconf.

To see why this extension failed to compile, please check the mkmf.log which can be found here:

  /usr/local/bundle/extensions/x86_64-linux/2.1.0-static/curb-0.9.3/mkmf.log

extconf failed, exit code 1

Gem files will remain installed in /usr/local/bundle/gems/curb-0.9.3 for inspection.
Results logged to /usr/local/bundle/extensions/x86_64-linux/2.1.0-static/curb-0.9.3/gem_make.out

DEBUG [2017-03-14 10:51:57.98]: uninitialized constant Fastlane::Badge
+-----------------------+-----------+------------------+
|                     Used plugins                     |
+-----------------------+-----------+------------------+
| Plugin                | Version   | Action           |
+-----------------------+-----------+------------------+
| fastlane-plugin-tpa   | 1.1.0     | tpa              |
| fastlane-plugin-badge | undefined | No actions found |
+-----------------------+-----------+------------------+

INFO [2017-03-14 10:51:58.00]: -------------------------------------------------
INFO [2017-03-14 10:51:58.01]: --- Step: Verifying required fastlane version ---
INFO [2017-03-14 10:51:58.01]: -------------------------------------------------
INFO [2017-03-14 10:51:58.01]: Your fastlane version 2.19.3 matches the minimum requirement of 2.0.5  ✅
```

**Comment by johnknapprs** [comment]
A new dependency was added apparently.  I got the same failures across our CI.  Just set your gemfile to the previous version

**Comment by waltermvp** [comment]
@johnknapprs do you have the latest version before the dependencies were added?

**Comment by PetrosIchor** [comment]
I dropped this plugin in favor of the fastlane official action awhile back.  I believe the version you want is `0.8.1`.  The latest (from march) is `0.8.2`, roll it back and see if that helps. 

Also, you can see the versions committed here:  https://github.com/HazAT/fastlane-plugin-badge/commits/master

**Comment by waltermvp** [comment]
Thanks but isn't it deprecated?

**Comment by PetrosIchor** [comment]
It's being deprecated.  Someone in their issues board made the argument that the plugin is better so the added the notice. 

If you compare, this plugin is much more limited.  It's out of date with the current plugin standards.  I'd argue the existing action is better simply for that reason. 

That aside, fastlane is also trying to downsize their actions and push the community to rely on plugins.  Trying to support 200+ features in an open source project is a steep hill to climb.  I get why they justify putting the notice up. 

If the plugin isn't working, you might consider creating your own action/plugin and simply copying then updating the existing action to meet in the middle.  You get published and the feature lives on a little longer.

**Comment by HazAT** [comment]
Hey, I've just released version `1.0.0` which renames the action to `add_badge`.
This plugin should support everything the original `badge`
https://github.com/HazAT/badge
is able to do. If not please open another issue.
